.github/workflows/ci.yml

```diff
@@ -49,10 +49,18 @@ jobs:
           ini-values: zend.assertions=1, error_reporting=-1, display_errors=On
           coverage: xdebug
 
-      # Install dependencies and handle caching in one go.
-      # @link https://github.com/marketplace/actions/install-composer-dependencies
+      - name: Determine composer cache directory
+        run: echo "COMPOSER_CACHE_DIR=$(composer config cache-dir)" >> $GITHUB_ENV
```

Comment:
Besides the fact that this will become unnecessary if we remove the caching entirely, it seems better to use the step output for that rather than creating a `COMPOSER_CACHE_DIR` env variable for next steps (especially given that `composer install` would be impacted by an env variable named `COMPOSER_CACHE_DIR`)

Reply:
@stof 

That's exactly the point of using [`COMPOSER_CACHE_DIR`](https://getcomposer.org/doc/03-cli.md#composer-cache-dir).

Reply:
but here, setting the env variable based on the existing cache dir is quite confusing

Reply:
@stof 

No, it's the most flexible.

These steps will work even if you configure the cache directory in `composer.json`.

Reply:
if you don't set a `COMPOSER_CACHE_DIR` env variable (configuring the cache dir for next composer usages), things are still fine when using step output to configure `actions/cache` as you **read** the cache dir from composer here.